src/utils/debugLogger.ts

```diff
@@ -34,7 +34,6 @@ class DebugLogger {
 
   constructor() {
     if (process.env.DEBUG_FILE) {
-      fs.mkdirSync(path.dirname(process.env.DEBUG_FILE), { recursive: true });
```

Comment:
does it always exist?

Reply:
We basically tell user that what they specify in DEBUG_FILE should point to an existing folder. And ourselves, we create it in the index.ts below.